src/flamenco/types/Makefile

```diff
@@ -0,0 +1,10 @@
+all:
+	@make -C ../../..
+
+clean:
+	@make -C ../../.. clean
+
+stubs: fd_types.json
+	python3 gen_stubs.py fd_types.h fd_types.c
+	uncrustify  -c lint.cfg --no-backup --replace fd_types.c
```

Comment:
It pains me but.. lets just remove the lint.cfg ...

Reply:
the white space will be a mess. what's wrong with lint?